rfcs/README.md

```diff
@@ -0,0 +1,27 @@
+# RFC's
+
+This folder contains RFC (Request for Comment) documents. A template can be found at `./_template.md`.
+
+RFCs are useful when team feedback is required before making architecture or design decisions, or any topic that warrants discussion facilitated with documentation.
+
+## Creating a new RFC
+
+- Create a new branch for your RFC PR
+- Copy the `_template.md` file to a new file that is prefixed with YYYY-MM & appropriately titled
+- Fill out the template accordingly
+- Submit the PR with the `[RFC]` PR title prefix and tag
+- Commence discussion, adjusting the RFC based on feedback
+- Once all questions defined in the RFC are satisfied, and PR is accepted, merge it
+
+### RFC Status Options
+
+- WIP: The author is still drafting the RFC and it’s not ready for review.
+- Review: The Review label is used when the RFC is ready for comments and feedback.
+- Approved: When the RFC is for the purpose of making a decision, the Approved label indicates that the decision has been made.
+- Implemented: When the RFC is for the purpose of making a decision, the Implemented label indicates that the RFC’s proposal has been implemented.
```

Comment:
I don't know for our use case if the `WIP` status is very useful because we have GitHub PR tags that overlap with that functionality, but I do not have a strong opinion. I think it might be something I personally would forget to change or just not use.

Reply:
On second thought, it might take longer to explain an alternative process of taggingn PRs and assigning reviewers than to just include this as a status, and people can use it or not.

Reply:
Do we update the date when the status changes?

Reply:
🤷🏻‍♀️ What is the benefit of updating the date in the doc vs referencing git history?
Additionally... is the date actually useful info?

Reply:
I think the date is useful just for understanding at a glance when it was started without looking at the log. For this reason, I don't think we'd update the date when changes are made—just keep it as the date of origination